guides/release/in-depth-topics/rendering-values.md

```diff
@@ -4,88 +4,129 @@ In Ember, rendering occurs via syntax, rather than by value -- so _anything_ can
+</template>
 ```
 
-Modifiers can be curried with the `modifier` helper:
+Modifiers can be curried with the `modifier` helper keyword:
 
-```handlebars
-{{! In a component called "Example" }}
-{{yield (modifier someModifier "arguments" "here")}}
+```gjs
+<template>
+  {{! In a component called "Example" }}
+  {{yield (modifier someModifier "arguments" "here")}}
+</template>
+```
 
-{{! Usage: }}
-<Example as |theModifier|>
-  <div {{theModifier}}>
-</Example>
+```gjs
+<template>
+  {{! Usage: }}
+  <Example as |theModifier|>
+    <div {{theModifier}}>
+  </Example>
+</template>
 ```
 
 
 ## Helpers
 
 For Helpers, there is a specific syntax that only helpers may reside in
-```handlebars
-{{ (theHelper) }}
+```gjs
+<template>
+  {{ (theHelper) }}
+</template>
 ```
 or nested in a sub-expression
-```handlebars
-{{yield (hash key=(theHelper) key2=(theHelper with args)) }}
+```gjs
+import { hash } from '@ember/helper';
+
+<template>
+  {{yield (hash key=(theHelper) key2=(theHelper with args)) }}
+</template>
 ```
 or via property on some object
-```handlebars
-{{ (this.property.theHelper) }}
+```gjs
+<template>
+  {{ (this.property.theHelper) }}
+</template>
 ```
 or via an argument passed to a component
-```handlebars
-{{ (@theHelper) }}
+```gjs
+<template>
+  {{ (@theHelper) }}
+</template>
 ```
 
-Helpers can be curried with the `helper` helper:
-```handlebars
-{{! In a component called "Example" }}
-{{yield (helper someHelper "arguments" "here")}}
+Helpers can be curried with the `helper` helper keyword:
+```gjs
+<template>
+  {{! In a component called "Example" }}
+  {{yield (helper someHelper "arguments" "here")}}
+</template>
+```
 
-{{! Usage: }}
-<Example as |theHelper|>
-  {{ (theHelper) }}
-</Example>
+```gjs
+<template>
+  {{! Usage: }}
+  <Example as |theHelper|>
+    {{ (theHelper) }}
+  </Example>
+</template>
 ```
 
 ## Components
 
 For Components, there is a specific syntax that only components may reside in
-```handlebars
-<AComponent />
+```gjs
+<template>
+  <AComponent />
+</template>
 ```
 or via a property on some object
-```handlebars
-<this.someComponent />
+```gjs
+<template>
+  <this.someComponent />
+</template>
 ```
 or via an argument passed to a component
-```handlebars
-<@someComponent />
+```gjs
+<template>
+  <@someComponent />
+</template>
 ```
 
-Components can be curried with the `component` helper:
-```handlebars
-{{!
+Components can be curried with the `component` helper keyword:
+```gjs
+<template>
+  {{!
   In a component called "Example".
   Note that components may only receive named arguments
-}}
-{{yield (component someComponent foo="arguments" bar="here")}}
+  }}
+  {{yield (component someComponent foo="arguments" bar="here")}}
+</template>
+```
 
-{{! Usage: }}
-<Example as |theComponent|>
-  <theComponent />
-</Example>
+```gjs
+<template>
+  {{! Usage: }}
+  <Example as |theComponent|>
+    <theComponent />
+  </Example>
+</template>
 ```
+<!-- eof - needed for pages that end in a code block  -->
```

Comment:
????????????

Reply:
It's for the tooling, apparently

Reply:
is this a new issue? this page has been deployed since ... sometime in 3.x, afaik?

Reply:
Not sure -- I just have seen it in the code elsewhere